sources/Launcher/DYNSystematicAnalysisLauncher.cpp

```diff
@@ -127,6 +127,9 @@ SystematicAnalysisLauncher::launchScenario(const boost::shared_ptr<Scenario>& sc
   SimulationResult result;
   result.setScenarioId(scenario->getId());
   boost::shared_ptr<DYN::Simulation> simulation = createAndInitSimulation(workingDir, job, params, result, inputs_);
+  simulation->setTimelineOutputFile("");
```

Comment:
Are you sure this is required?  The stream is filled even if it was dumped in a file no?

Reply:
yes this is precisely the interest of the dev, before the file was written twice

Reply:
Are we sure this has no effect on the platform?

Reply:
the platform read the files from the zip so the local files are never used